<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>10</y>
     <width>501</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border-radius:20px;</string>
   </property>
   <widget class="QLineEdit" name="identifiant_field">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background:#ffeeef;
border:2px solid #eb2f30;
border-radius:10px;
color:black;
}
QLineEdit:focus
{
border:2px solid #FBAD25;

}</string>
    </property>
    <property name="placeholderText">
     <string>Enter Identifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_field">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background:#ffeeef;
border:2px solid #eb2f30;
border-radius:10px;
color:black;
}
QLineEdit:focus
{
border:2px solid #FBAD25;

}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Enter password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>361</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#register_btn
{
background-color :#eb2f30;  
color :white;
border-radius : 5px;

}

QPushButton#register_btn:pressed
{
background-color :#de5758;  
}</string>
    </property>
    <property name="text">
     <string>R e g i s t e r</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_in_btn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#log_in_btn
{
background-color :#eb2f30;  
color :white;
border-radius : 5px;

}

QPushButton#log_in_btn:pressed
{
background-color :#de5758;  
}</string>
    </property>
    <property name="text">
     <string>Log In </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Óùª</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_field">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background:#ffeeef;
border:2px solid #eb2f30;
border-radius:10px;
color:black;
}
QLineEdit:focus
{
border:2px solid #FBAD25;

}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phonenumb_field">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background:#ffeeef;
border:2px solid #eb2f30;
border-radius:10px;
color:black;
}
QLineEdit:focus
{
border:2px solid #FBAD25;

}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Enter phone number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name_field">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background:#ffeeef;
border:2px solid #eb2f30;
border-radius:10px;
color:black;
}
QLineEdit:focus
{
border:2px solid #FBAD25;

}</string>
    </property>
    <property name="placeholderText">
     <string>Enter Full Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color :black</string>
    </property>
    <property name="text">
     <string>Register Form</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color :black  </string>
    </property>
    <property name="text">
     <string>Already having an account ? </string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>471</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#d3d3d3;
border-radius:20px;</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>211</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>design/icons/esprit.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
